src/modules/a11y/a11y.js

```diff
@@ -131,9 +131,10 @@ export default function A11y({ swiper, extendParams, on }) {
 
   function updatePagination() {
     const params = swiper.params.a11y;
-    if (hasPagination()) {
-      swiper.pagination.bullets.each((bulletEl) => {
-        const $bulletEl = $(bulletEl);
+    const doHasPagination = hasPagination();
```

Comment:
Maybe there is a sense to replace this line with just 
```
if (!hasPagination()) return;
```
I mean what is the point to to loop over `swiper.pagination.bullets` and do checks for pagination inside if `hasPagination()` return false?

Reply:
`hasPagination()` checks for `clickable`, that not necessary for aria-current
```js
function hasPagination() {
    return (
      swiper.pagination &&
      swiper.params.pagination.clickable &&
      swiper.pagination.bullets &&
      swiper.pagination.bullets.length
    );
  }
```

Reply:
Then it should be tweaked, otherwise if pagination type is other than bullets then more likely `swiper.pagination.bullets.each` will throw an error

Reply:
Take a look on latest commit